manta/manta.go

```diff
@@ -191,8 +191,8 @@ func (c *Client) GetObject(path, objectName string) ([]byte, error) {
 	if err != nil {
 		return nil, errors.Newf(err, "failed to get object %s/%s", path, objectName)
 	}
-	res, _ := respData.RespValue.([]byte)
-	return res, nil
+	res, _ := respData.RespValue.(*[]byte)
```

Comment:
That's a good thought. I originally thought that might be a poor experience for the users of the library since the code would be failing without their intervention or fault, but I suppose it is always helpful to be more verbose and report errors than just to return empty data. See my new commit.